hw/ip/edn/doc/theory_of_operation.md

```diff
@@ -15,6 +15,8 @@ Whenever commands are directly forwarded from firmware to the CSRNG through the
 `CMD_RDY` indicates whether the EDN is ready to receive a new command, whereas `CMD_REG_RDY` indicates whether the EDN is ready to accept the next word of the command.
 Each command consists of a command header, which is represented by a single word.
 Furthermore, a command can have up to 12 words of additional data.
+(In case firmware erroneously configures a `clen` value greater than 12, the EDN hardware expects it to indeed provide the specified number words of additional data.
+CSRNG itself will only consider the first 12 words of additional data and silently discard the rest.)
```

Comment:
Grammar nit: I'd probably tweak this to "... and will silently discard ... " because I think it reads more cleanly. I'm not *certain* that the text here was wrong though, so maybe it doesn't matter :-)